webroot/src/components/SelectedStatePurchaseInformation/SelectedStatePurchaseInformation.less

```diff
@@ -0,0 +1,111 @@
+//
+//  SelectedStatePurchaseInformation.less
+//  CompactConnect
+//
+//  Created by InspiringApps on 11/12/2024.
+//
+.selected-state-purchase-info-container {
+    width: 27rem;
+    margin: 0.4rem 0;
+    padding: 1rem 1.6rem;
+    border-radius: 20px;
+    color: white;
+    background-color: @primaryColor;
+
+    @media @tabletWidth {
+        margin: 1rem 1rem;
+    }
+
+    .jurisprudence-modal {
```

Comment:
I'm seeing the modal be 100% width on mobile. From the designs, it looks like it should be more like ~90% (ish).

<img width="512" alt="Screenshot 2024-11-20 at 10 55 28 AM" src="https://github.com/user-attachments/assets/c4d48268-c0f1-47e6-a54a-3a7f095ced51">